networkx/algorithms/min_edge_cover.py

```diff
@@ -0,0 +1,74 @@
+"""
+An edge cover of a graph is a set of edges such that every vertex of
+the graph is incident to at least one edge of the set.
+A minimum edge cover is an edge covering of smallest possible size.
+
+https://en.wikipedia.org/wiki/Edge_cover
```

Comment:
Don't leave raw links in user-facing documentation. Use [restructured text format][1] to provide meaningful link titles.

[1]: http://sphinx-doc.org/rest.html